packages/actor-rdf-dereference-paged-next/test/MediatedPagedAsyncRdfIterator-test.ts

```diff
@@ -7,7 +8,6 @@ import {
 } from '@comunica/bus-rdf-metadata-extract';
 import { ActionContext, IActorTest, Mediator } from '@comunica/core';
 import { SingletonIterator } from 'asynciterator';
-import * as _ from 'lodash';
```

Comment:
Actually, this one can stay (don't forget to re-add it in the root package.json as well), since we're using it as a devDependency in our tests.

Reply:
In the root package.json only 1 line was deleted in this PR and that's a lodash.assign dependency. Not sure if you mean this one?